webpack/components/AnsibleHostDetail/AnsibleHostDetail.scss

```diff
@@ -0,0 +1,6 @@
+@import '~@patternfly/patternfly/base/patternfly-variables';
```

Comment:
doesn't it a list of PF3 variables?

Reply:
[foreman-js](https://github.com/theforeman/foreman-js/blob/master/packages/vendor-core/scss/variables.scss#L20) says these are pf4...